.github/workflows/run-mill-action.yml

```diff
@@ -36,8 +36,8 @@ jobs:
 
       - name: Prepare git config
         run: |
-          git config --add user.name "Mill GithHub Actions"
-          git config --add user.email "mill-ci@localhost"
+          git config --global user.name "Mill GithHub Actions"
+          git config --global user.email "mill-ci@localhost"
```

Comment:
Some of the example tests use git in a newly created repo in a sub-folder, so need the name and email to be set globally